finance/funding_requests.md

```diff
@@ -26,7 +26,7 @@ The Finance Committee will aim to provide approval within 1 week of the request.
 
 ### After Travel
 
-Wthin two weeks of completing your trip you should submit your reimbursment request via the [NumFOCUS Rocket form | https://numfocus.typeform.com/to/oTBLP2] You will need to include itemized receipts for all travel costst. You will also need to provide information for your prefered payment method i.e. back account and routing number, paypal account, etc. 
+Within two weeks of completing your trip you should submit your reimbursement request via the [NumFOCUS Rocket form | https://numfocus.typeform.com/to/oTBLP2] You will need to include itemized receipts for all travel costs. You will also need to provide information for your preferred payment method i.e. back account and routing number, paypal account, etc. 
```

Comment:
Is this 2 weeks here realistic while receiving the reimbursement takes 30 days?

Reply:
At my institution (MIT) we have to submit reimbursement requests within 3 days of returning and payment typically takes longer than 30 days. 
So, this is not unheard of. The point is that most astronomers do things on the day of the deadline, whenever that deadline is. Note the word "should" - you *should* submit within two weeks, but Astropy will reimburse you if it takes you longer (within reason).

Reply:
Yes, I noted the word, thus commented. In the previous iteration it was "preferably" which is noticably a more friendly way than "should". MIT's probably the most extreme case, and since we don't follow their policies in other questions what about being more flexible with the wording here, too? No one stops anyone to follow their home institute's shorter deadline policies and submit the reimbursement in 3 days, but in most cases that deadline is more in the 1-2 months range anyway.